AraEvent/AraEventCalibrator.cxx

```diff
@@ -1034,8 +1034,10 @@ void AraEventCalibrator::calibrateEvent(UsefulAtriStationEvent *theEvent, AraCal
         ( similar to ARA5. So, adding  && thisStationId != 4 to the following condition so that we "DO NOT" zero mean before Voltage calibration. 
         In the future, we might need to check if A4 also has outlier event or not, but for now, we are not zero-meaning before v calib.
         latest update : 29th Nov2022, PDG is debugging ! 
+    
+        Station A1 followed same calibration procedure as A4 and A5. Not applying zero-mean for A1 (stationID 100 or 1) --Mohammad
     */ //do not zero mean if station==4, station==5 
-    if(hasADCZeroMean(calType) && thisStationId != 5 && thisStationId != 4) {
+    if(hasADCZeroMean(calType) && thisStationId != 5 && thisStationId != 4 && thisStationId != 100 && thisStationId != 1) {
```

Comment:
For this if statement and a few others in this file: does the calibration really apply to the ICRR station? ie `stationId == 1`?

Reply:
I kept "stationId == 1" to be on a safe side. Some of the later A1 runs are contaminated with very early 2011 runs.

Reply:
I think we should not do this. If those runs are really ICRR we'd be using the wrong voltage calibration silently. If no ICRR calibration exists then I think it's better that the code crashes than to let it use the ATRI calibration silently. That will also help us identify any runs that are in the wrong directories and correct that.

Reply:
Okay, I will remove "stationId == 1".

Reply:
Thanks!